packages/api-server/src/watch.ts

```diff
@@ -119,19 +120,19 @@ const rebuildApiServer = () => {
     // Check if experimental server file exists
     const serverFile = resolveFile(`${rwjsPaths.api.dist}/server`)
     if (serverFile) {
-      const separator = chalk.hex('#ff845e')(
-        '------------------------------------------------------------------'
-      )
+      const separator = chalk.hex('#ff845e')('-'.repeat(79))
       console.log(
         [
           separator,
           `🧪 ${chalk.green('Experimental Feature')} 🧪`,
           separator,
-          'Using the experimental API server file at api/dist/server.js',
+          'Using the experimental API server file at api/dist/server.js (in watch mode)',
```

Comment:
not something you introduced but looks like the extension is hardcoded here; no pressure to fix just calling it out

Reply:
It is, and I did see that. Since it's pointing to `/dist/` it's always going to be `.js`, so I think it's correct